assets/js/components/setup/setup-proxy.js

```diff
@@ -87,6 +87,24 @@ class SetupUsingProxy extends Component {
 			siteHostname,
 		} = this.state;
 		const isRevoked = 'revoked' === context;
+		const isSecondAdmin = ! isRevoked && isResettable;
+
+		let title = __( 'Sign in with Google to set up Site Kit', 'google-site-kit' );
+		let description = __( 'The Site Kit service will guide you through 3 simple setup steps.', 'google-site-kit' );
+		let startSetupText = __( 'Start setup', 'google-site-kit' );
+
+		if ( isSecondAdmin ) {
+			title = __( 'Sign in with Google to configure Site Kit' );
+			description = __( 'To use Site Kit, sign in with your Google account. Site Kit Service will guide you through 3 simple steps to complete the connection and configure the plugin.' );
+			startSetupText = __( 'Sign in with Google', 'google-site-kit' );
+		} else if ( isRevoked ) {
+			title = sprintf(
+				/* translators: %s is the site's hostname. (e.g. example.com) */
+				__( 'You revoked access to Site Kit for %s', 'google-site-kit' ),
+				siteHostname
+			);
+			description = __( 'Site Kit will no longer have access to your account. If you’d like to reconnect Site Kit, click "Start Setup" below to generate new credentials.', 'google-site-kit' );
+		}
```

Comment:
Just a recommendation to simplify here: We could turn around the if elseif checks to check `isRevoked` first. This way we don't need the `isSecondAdmin` assignment in line 90 and could just rely on `isResettable`.

Reply:
This simplification would logically be the same, but is likely less clear/self-documenting at the same time. Not saying we shouldn't do it, but probably also worth adding a comment as to what that section is for as there would no longer be a clear `isSecondAdmin` to label/describe that case.

Reply:
I agreed with @aaemnnosttv—that's why I created the extra variable and coded this as it is. It's harder to follow the intent of the code without the `isSecondAdmin` variable. If we changed the code around to remove it we'd need to add comments to document it and the logic would be less clear in the code, so I think it's better to leave it as-is.

Reply:
That's a fair point. I'd argue that it would still make sense to turn around the order though. `isSecondAdmin` is not actually dependent on whether access was not revoked. We only want the revoke message to take precedence. So my suggestion is to turn around the conditions and then simply assign `isResettable` to `isSecondAdmin`. That keeps the intentions clear and the code more simple.

Reply:
Changes made—let me know if that looks good or if you want any further changes 😄